mods/taskbar-clock-customization.wh.cpp

```diff
@@ -507,6 +554,13 @@ HANDLE g_webContentUpdateStopEvent;
 std::mutex g_webContentMutex;
 std::atomic<bool> g_webContentLoaded;
 
+// Media session variables
+winrt::Windows::Media::Control::GlobalSystemMediaTransportControlsSessionManager g_mediaSessionManager{ nullptr };
+std::atomic<bool> g_mediaSessionInitialized;
```

Comment:
Instead of having this additional bool, you can simply check whether `g_mediaSessionManager` is null.